.github/workflows/release.yaml

```diff
@@ -46,6 +46,18 @@ jobs:
       with:
         node-version: 14.x
 
+    - name: Download wix toolset
+      if: matrix.os == 'windows-latest'
+      run: curl -OLS https://github.com/wixtoolset/wix3/releases/download/wix3111rtm/wix311.exe
+    
+    - name: Install wix toolset
+      if: matrix.os == 'windows-latest'
+      run: .\wix311.exe /install /norestart
+
+    - name: Add wix toolset binaries to PATH
+      if: matrix.os == 'windows-latest'
+      run: echo "C:\Program Files (x86)\WiX Toolset v3.11\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+      
```

Comment:
With a multiline run command this can be written as a single "Install wix toolset" step.